proof/invariant-abstract/ARM_HYP/ArchArch_AI.thy

```diff
@@ -910,7 +910,7 @@ lemma associate_vcpu_tcb_sym_refs_hyp[wp]:
        apply (clarsimp simp add: sym_refs_def)
        apply (case_tac "x = t"; case_tac "x = vr"; clarsimp simp add: state_hyp_refs_of_def obj_at_def
                                                                  dest!: get_tcb_SomeD)
-       apply fastforce
+        apply fastforce
       apply (rule hoare_pre)
```

Comment:
Is this one intentional? Looks inconsistent with the `hoare_pre` line.

Reply:
I'll doube check this one